### ISSUE TITLE
Pkg.submit() not working on Windows

### ISSUE THREAD

**Issue by KrisDM** [JuliaLang/julia#14813]
Working with Julia 0.4.3 on Windows. 

I made some changes to a package (on my own branch, as suggested in the documentation). 

I did a commit, filled out a commit message in vim and after saving that and attempting to submit I get the following:
```
julia> Pkg.submit("PDMats")
INFO: Forking JuliaStats/PDMats.jl to KrisDM
ERROR: could not spawn `curl --version`: no such file or directory (ENOENT)
 in _jl_spawn at process.jl:262
```
I was asked to open an issue here after posting originally in Pkdev.jl: https://github.com/JuliaLang/PkgDev.jl/issues/26

**Comment by tkelman** [comment]
Thanks. I'll test and push an 0.4 specific fix to the release branch for this issue. On nightly the fix will be different and happen outside of base, in PkgDev.jl and possibly a new PowerShell.jl package if the code is worth separating out.

There's a workaround you can do for now of temporarily modifying your path, though it does break starting julia in mintty. Try `ENV["PATH"] *= ";" * joinpath(JULIA_HOME,"..","Git","usr","bin")` for now.

**Comment by KrisDM** [comment]
Thanks for the workaround. I've managed to get git push to work from julia, which works fine for now.

**Comment by tkelman** [comment]
PR at #14849, will be fixed for 0.4.4.
